sailfish-types/src/message.rs

```diff
@@ -247,6 +251,10 @@ impl<T: Committable> Payload<T> {
         self.data
     }
 
+    pub fn evidence(&self) -> &Evidence {
+        &self.evidence
+    }
+
     pub fn into_parts(self) -> (RoundNumber, PublicKey, T) {
```

Comment:
```suggestion
    pub fn into_parts(self) -> (RoundNumber, PublicKey, T, Evidence) {
```
A separate `into_evidence` method could be useful as well.